docs/package.json

```diff
@@ -55,7 +55,7 @@
     "ember-qunit": "7.0.0",
     "ember-resolver": "13.1.1",
     "ember-sinon-qunit": "7.5.0",
-    "ember-source": "5.12.0",
+    "ember-source": "6.7.0",
```

Comment:
Upgraded to Ember v6.7.0 only due to regression in Ember v6.8 which breaks Fastboot. The docs app use Fastboot to prerender the pages.